docs/source/AdministratorGuide/Systems/Transformation/index.rst

```diff
@@ -217,7 +217,11 @@ Configuration
 Plugins
 -------
 
-There are two different types of plugins, i.e. TransformationAgent plugins and TaskManager plugins. The first are used to 'group' the input files of the tasks according to different criteria, while the latter are used to specify the tasks destinations.
+There are three different types of plugins:
+
+* TransformationAgent plugins: these are used to 'group' the input files of the tasks according to different criteria.
+* TaskManager plugins: these are used only by the ``WorkflowTasks`` to specify the tasks destinations.
+* BodyPlugins: these are used at the moment only by the ``RequestTasks`` and allow to have a complex logic of transforming a task into an RMS Request
```

Comment:
Could they potentially be used also for creating WMS requests (jobs)?

Reply:
Hum, if the `WorkflowTasks` is instrumented to use them, maybe ! But I don't know this part at all